### PR TITLE
TLS/SSL Updates

### PR DESCRIPTION
**Pull request by kagarmoe**
## Description

While working on the mTLS docs, I noticed that lot of our SSL docs are a bit dated. This PR is for clarification & making them more current.

## Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/chef_client_security.md [hunk 1]
```diff
@@ -47,118 +47,85 @@ knife[:authentication_protocol_version] = '1.3'
...
 versions 12.4.0 and above.
 
-## SSL Certificates
+## TLS/SSL Certificates
 
 {{< warning >}}
 
-The following information does not apply to hosted Chef Server 12, only
-to on-premises Chef Server 12.
+The following information does not apply to hosted Chef Server 12
```
Comment (kagarmoe) [inline content/chef_client_security.md:54]: Does it apply to any Hosted Chef?

Comment (tas50) [reply content/chef_client_security.md:54]: That should probably be "Hosted Chef"

File: content/chef_repo.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ to contain cookbook-specific copyright, email, and license data.
 The `data_bags/` directory is used to store all of the data bags that
 exist for an organization. Each sub-directory corresponds to a single
 data bag on the Chef Infra Server and contains a JSON file for each data
-bag item. If a sub-directory does not exist, then create it using SSL
+bag item. If a sub-directory does not exist, then create it using TLS/SSL
```
Comment (kagarmoe) [inline content/chef_repo.md:71]: @tas50 I don't understand this--is it wrong? How does one create a data bag with SSL commands?
```suggestion
bag item. If a sub-directory does not exist, then create it using SSL
```

**Comment by kagarmoe** [comment]
Close until this work can be scheduled

**Pull request closed without merging**